interoperability-tests/Testing-between-Gramine-and-IntelSGXSDK-ratls.md

```diff
@@ -0,0 +1,158 @@
+Testing between Gramine and rats-tls
+====
+
+# Introduction
+
+This document records the interoperability testing between [Gramine](https://github.com/gramineproject/gramine) and [Intel SGXSDK](https://github.com/intel/linux-sgx), both of which have implemented Interoperable RA-TLS.
+
+We conducted two sets of tests, in which Gramine and Intel SGXSDK ra-tls respectively verified each other's certificates and established TLS secure channels. You can verify our tests by following the steps below.
```

Comment:
`... SGX SDK ra-tls ...` -- please use capital letters: `SGX SDK RA-TLS`. The official name of RA-TLS is with all capital letters, see e.g. the Release Notes: https://github.com/intel/linux-sgx/releases/tag/sgx_2.22

Here and everywhere.

Reply:
updated